conans/client/command.py

```diff
@@ -1000,8 +1008,29 @@ def search(self, *args):
         info = None
 
         try:
-            if ref:
+            if not v2_blocked and args.revisions:
```

Comment:
Wouldn`t this raise when the evaluation of ``args.revisions`` is done and ``v2_blocked=False`` ?